pkgs/misc/drivers/epson-201202w/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib, stdenv, fetchurl, rpmextract, autoreconfHook, file, libjpeg, cups }:
+
+let
+  version = "1.0.0";
```

Comment:
generally, it is preferred to use `stdenv.mkDerivation {` and `version = "1.0.0";` directly

Reply:
Thanks, I was following the most similar derivation already in the repo I could find, but I am very noob with nix, so I am not sure what you mean. Feel free to make changes if you are more confident